source/fuzz/fuzzer_pass_add_equation_instructions.cpp

```diff
@@ -21,6 +21,26 @@
 
 namespace spvtools {
 namespace fuzz {
+namespace {
+
+bool IsBitWidthSupported(opt::IRContext* ir_context, uint32_t bit_width) {
+  switch (bit_width) {
+    case 32:
+      return true;
+    case 64:
+      return ir_context->get_feature_mgr()->HasCapability(
+                 SpvCapabilityFloat64) &&
+             ir_context->get_feature_mgr()->HasCapability(SpvCapabilityInt64);
+    case 16:
+      return ir_context->get_feature_mgr()->HasCapability(
+                 SpvCapabilityFloat16) &&
+             ir_context->get_feature_mgr()->HasCapability(SpvCapabilityInt16);
```

Comment:
I guess you could add the 8-bit integer type case.
```
case 8:
  return ir_context->get_feature_mgr()->HasCapability(SpvCapabilityInt8);
```

Reply:
We don't have `Float8` capability so it seems we can't create floats of 8 bits. Thus, I think we can't support it.

Reply:
But I am saying to add support for 8-bit _integer_ type.